modules/consensus/accept.go

```diff
@@ -179,9 +180,19 @@ func (cs *ConsensusSet) AcceptBlock(b types.Block) error {
 	if len(changeEntry.AppliedBlocks) > 0 {
 		cs.readlockUpdateSubscribers(changeEntry)
 	}
+	return nil
+}
 
+// AcceptBlock will add a block to the state, forking the blockchain if it is
+// on a fork that is heavier than the current fork. If the block is accepted,
+// it will be relayed to connected peers. This function should only be called
+// for new, untrusted blocks.
```

Comment:
Wording is not super clear. I realize it's what I wrote, but today I'd probably rewrite it.

```go
// AcceptBlock will try to add a block to the consensus set. If the block does not extend the longest currently known chain, an error is returned but the block is still kept in memory. If the block extends a fork such that the fork becomes the longest currently known chain, the consensus set will reorganize itself to recognize the new longest fork. If a block is accepted without error, it will be relayed to all connected peers. This function should only be called for new blocks.
```